src/unix/poll.c

```diff
@@ -33,8 +33,21 @@ static void uv__poll_io(uv_loop_t* loop, uv__io_t* w, unsigned int events) {
 
   handle = container_of(w, uv_poll_t, io_watcher);
 
+#ifdef __linux__
+  /*
+   * As documented in the kernel source fs/kernfs/file.c #780
+   * poll will return POLLERR|POLLPRI in case of sysfs
+   * polling. This does not happen in case of out-of-bound
+   * TCP messages.
+   *
+   * So to properly determine a POLLPRI or a POLLERR we need
+   * to check for both.
+   */
+  if ((events & POLLERR) && (events & UV__POLLPRI) != UV__POLLPRI) {
+#else
```

Comment:
Maybe I'm missing something obvious, but isn't this the same as:
`if ((events & POLLERR) && !(events & UV__POLLPRI)) {`